### ISSUE TITLE
Matriz de avance

### ISSUE THREAD

**Issue by jordicarsan** [darktable-org/darktable#18060]
### Describe the bug

At time ti import to the library. I don't know what it mean

### Steps to reproduce

When archives are importing the monitor shows a text and says report to darktable by Github

### Expected behavior

I don't know

### Logfile | Screenshot | Screencast

![imatge](https://github.com/user-attachments/assets/b290e6d0-2a31-4948-9a68-facb57686cab)

### Where did you obtain darktable from?

OBS

### darktable version

4.5.0

### What OS are you using?

Linux

### What is the version of your OS?

Debian sid

### Describe your system?

i7 - 8565U 16 GB RAM GPU Intel WhyskeyLake-u  GT2 GPU Nvidia 1650 Mobile  driver noveau

### Are you using OpenCL GPU in darktable?

None

### If yes, what is the GPU card and driver?

GPU Intel WhyskeyLake-u  GT2 GPU Nvidia 1650 Mobile  driver noveau

### Please provide additional context if applicable. You can attach files too, but might need to rename to .txt or .zip

I use raw file . ORF generally but I have a lot of diferent raw formats and also DNG

**Comment by piratenpanda** [comment]
1. Why are you using a really old version 
2. Which camera is the dng file from
3. Could you upload one of the dngs somewhere?

**Comment by jordicarsan** [comment]
1, is the last version. 5.0
2. Maybe the Fujifilm xpro2. I m not sure.
3. I try after Christmas 
-- 
 Enviat amb Tuta, la bústia segura i lliure d'anuncis. 



24 de des. del 2024, 23:16 per ***@***.***:

**Comment by geni1105** [comment]
I get the same error message with darktable 5.0 on macOS 14.6 Sonoma, when importing DNGs generated by Kalpanika (from Sigma DP Merrill X3Fs) or directly from a Sigma sd quattro H, i.e. linear DNGs. 
All can be processed just fine, though. 

Other DNGs with a Bayer pattern, like from Sigma fp L oder Leica D-Lux 8 (sample images downloaded from dpreview.com), do not to trigger that error message.

**Comment by victoryforce** [comment]
> When archives are importing the monitor shows a text and says report to darktable by Github

Yes, and I'm closing the issue. In 5.2 this message will not be displayed in a case like yours (when importing a dng obtained by converting the original camera raw, and not a dng created by the camera itself).
